cmd/cstorvolumeclaim/cstorvolumeclaim.go

```diff
@@ -208,11 +253,10 @@ func createTargetService(storageClassName string,
 	if k8serror.IsNotFound(err) {
 		svcObj, err = svc.NewBuilder().
 			WithName(claim.Name).
-			WithLabelsNew(labels).
-			WithAnnotations(annotations).
```

Comment:
this is missing

Reply:
We don't  any required annotations for service yet